### PR TITLE
Video 10586 synk pr

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDE0-10586](https://issues.corp.twilio.com/browse/VIDE0-10586)

### Description

This PR updates the following dependencies to the versions listed below:
- react-scripts: `5.0.0`
- @types/jest: `^27.5.2`
- ts-jest: `^27.0.1`
- firebase: `^9.9.0`

Note that there were additional jest configurations needed in order to test code associated with the SwiperJs library. Please see [this solution](https://stackoverflow.com/questions/72141568/swiper-8-and-jest/72926658#72926658:~:text=I%20have%20the%20same%20issue%20and%20still%20searching%20for%20a%20solution%3B%20I%20believe%20what%20the%20OP%20is%20asking%20is%20how%20can%20we%20transform%20swiper/react%20into%20a%20CJS%20module%20on%20the%20JEST%20pipeline.) for more information.

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: src/components/AboutDialog/AboutDialog.tsx [hunk 1]
```diff
@@ -25,7 +24,7 @@ function AboutDialog({ open, onClose }: PropsWithChildren<AboutDialogProps>) {
       <DialogContent>
         <DialogContentText>Browser supported: {String(Video.isSupported)}</DialogContentText>
         <DialogContentText>SDK Version: {Video.version}</DialogContentText>
-        <DialogContentText>App Version: {appVersion}</DialogContentText>
+        <DialogContentText>App Version: {process.env.REACT_APP_VERSION}</DialogContentText>
```
Comment (timmydoza) [inline src/components/AboutDialog/AboutDialog.tsx:27]: Here I wonder what we should have when we run `npm start`?
Right now it just looks like `App Version:` with nothing after it. Should we remove it altogether? Or say "See package.json"? Or should we populate REACT_APP_VERSION when we run `npm start`? What do you think?

File: jest.config.js [hunk 2]
```diff
@@ -13,5 +16,9 @@ module.exports = {
   coveragePathIgnorePatterns: ['node_modules', 'src/icons'],
   moduleNameMapper: {
     '.+\\.(css|styl|less|sass|scss|png|jpg|ttf|woff|woff2)$': '<rootDir>/src/__mocks__/fileMock.ts',
+    "swiper/react": "<rootDir>/node_modules/swiper/react/swiper-react.js",
+    "swiper/css": "<rootDir>/node_modules/swiper/swiper.min.css",
+    "swiper/css/bundle": "<rootDir>/node_modules/swiper/swiper-bundle.min.css",
```
Comment (timmydoza) [inline jest.config.js:21]:
```suggestion
```

Don't think this one is needed.

File: scripts/build.js [hunk 3]
```diff
@@ -40,6 +40,7 @@ class Filter extends Transform {
 
 // Colors normally don't work when using spawn(), so here we re-enable colors.
 process.env.FORCE_COLOR = require('supports-color').stdout.level;
+process.env.REACT_APP_VERSION = require(__dirname + "/../package.json").version
```
Comment (timmydoza) [inline scripts/build.js:43]: Github won't let me comment on this, but up on line 20, the URL needs to be updated to:
```
  https://cra.link/deployment
```

**Review by timmydoza — CHANGES_REQUESTED** [review]

File: src/components/AboutDialog/AboutDialog.tsx (hunk 1 above)
Comment (olipyskoty) [reply src/components/AboutDialog/AboutDialog.tsx:27]: I added it to `npm start`. I thought about it and figured it is an easy way to check the app version so why not. It was also an easy addition 😄 . On the other hand, I suppose getting rid of it altogether isn't a bad idea either..

File: src/components/ChatWindow/MessageList/__snapshots__/MessageList.test.tsx.snap [hunk 4]
```diff
@@ -156,77 +156,7 @@ exports[`the messageList component should render correctly 1`] = `
...
-            >
-              test1.txt
-            </p>
-            <p
-              class="makeStyles-size-13"
-            >
-              120.56 KB
-               - Click to open
-            </p>
-          </div>
```
Comment (timmydoza) [inline src/components/ChatWindow/MessageList/__snapshots__/MessageList.test.tsx.snap:228]: Where did this message go? Should it still be there? I think MessageList.test needs to be updated to handle the new `media` type.

**Review by timmydoza — CHANGES_REQUESTED** [review]

**Review by timmydoza — APPROVED** [review]

**Pull request merged**